hazelcast-jet-hadoop/src/main/java/com/hazelcast/jet/hadoop/impl/ReadHdfsNewApiP.java

```diff
@@ -139,6 +140,7 @@ private static InputFormat getInputFormat(Configuration configuration) {
 
         static final long serialVersionUID = 1L;
 
+        @SuppressFBWarnings("SE_BAD_FIELD")
```

Comment:
what's the reason for this?

Reply:
It requires non serializable classes to be marked as transient, but the instances of the subclasses used here is always `Serializable`.

Reply:
ok, might be good to add the reasoning as comments